src/main/java/automation/pages/Utils.java

```diff
@@ -17,8 +19,12 @@ public static void selectFromList(WebElement element,String text) {
 	
 public static void scrollbyY(WebElement elementlocator) {
 	WebElement ele=driver.findElement((By) elementlocator);
-	((JavascriptExecutor)driver).executeScript("window.scrollTo(0,"+ele.getLocation().y+")");
+	((JavascriptExecutor)driver).executeScript("window.scrollTo(0," + ele.getLocation().y + ")");
 	
 }
+	public static void waitForElementVisible(WebElement element, int time) {
+		WebDriverWait wait = new WebDriverWait(driver,time);
+		wait.until(ExpectedConditions.visibilityOf(element));
```

Comment:
can you create a variable WebDriverWait wait outside the method, so that you can use it in other methods too